JeMPI_Apps/JeMPI_LibAPI/src/main/java/org/jembi/jempi/libapi/ProxyRoutes.java

```diff
@@ -458,43 +458,59 @@ static Route proxyPatchCrUpdateFields(
                     });
    }
 
-   private static CompletionStage<HttpResponse> proxyGetCandidatesWithScoreDoIt(
-         final String linkerIP,
-         final Integer linkerPort,
-         final Http http,
-         final String iid) {
-      final var uri = String.format(Locale.ROOT,
-                                    "http://%s:%d/JeMPI/%s?iid=%s",
-                                    linkerIP,
-                                    linkerPort,
-                                    GlobalConstants.SEGMENT_PROXY_GET_CANDIDATES_WITH_SCORES,
-                                    iid);
-      final var request = HttpRequest.create(uri);
-      final var stage = http.singleRequest(request);
-      return stage.thenApply(response -> response);
+   private static CompletionStage<HttpResponse> proxyPostCandidatesWithScoreDoIt(
+        final String linkerIP,
+        final Integer linkerPort,
+        final Http http,
+        final String iid) {
+    // Construct the URI without query parameters
+         final var uri = String.format(Locale.ROOT,
+                  "http://%s:%d/JeMPI/%s",
+                  linkerIP,
+                  linkerPort,
+                  GlobalConstants.SEGMENT_PROXY_POST_CANDIDATES_WITH_SCORES);
+
+         // Construct the request body with the iid parameter
+         final var json = "{\"uid\": \"" + iid + "\"}";
```

Comment:
@sushantpatil1214 use this implementation eg json = OBJECT_MAPPER.writeValueAsBytes(body);